src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/Services/TagHelperSemanticRangeVisitor.cs

```diff
@@ -19,7 +20,7 @@ internal class TagHelperSemanticRangeVisitor : SyntaxWalker
         private readonly RazorCodeDocument _razorCodeDocument;
         private readonly Range? _range;
 
-        private TagHelperSemanticRangeVisitor(RazorCodeDocument razorCodeDocument, Range? range)
+        private TagHelperSemanticRangeVisitor(RazorCodeDocument razorCodeDocument, Range? range, TextSpan? rangeAsTextSpan) : base(rangeAsTextSpan)
```

Comment:
I don't think I knew this base constructor existed. What changes as a result of passing `rangeAsTextSpan` in?